apps/storybook/src/Table.stories.tsx

```diff
@@ -92,7 +92,7 @@ export const Basic = () => {
       },
     ],
     [],
-  );
+  ) satisfies Column<{ name: string; description: string }>[];
```

Comment:
this i dont understand. so everyone will get an error if their columns don't have explicit types set?

Reply:
Not always, but yes, in most cases users might need to use `satisfies` or `as` to specify a type for props that they pass to `<Table>`. Sometimes, the generic type can be omitted.

```diff
- const columns = [...] satisfies Column<{ name: string, description: string }>[]
+ const columns = [...] satisfies Column[]
```

Reply:
lets show the less verbose version then

Reply:
```suggestion
  ) satisfies Column[];
```

Reply:
https://github.com/iTwin/iTwinUI/pull/1570/commits/31623868a5a2b50d0fe40f0545cbc68b448e8188: I implemented the above change and also removed any unnecessary type assertions. I tried to show examples that are as type lenient as possible. I also replaced a few type assertions to use `satisfies` instead.